boundary_layer_default_plugin/config/operators/dataproc_cluster_delete.yaml

```diff
@@ -20,15 +20,14 @@ parameters_jsonschema:
     properties:
         cluster_name:
             type: string
+        project_id:
+            type: string
         region:
             type: string
-        priority_weight:
-            type: integer
-            default: 50
-        trigger_rule:
+        gcp_conn_id:
+            type: string
+        delegate_to:
             type: string
-            default: all_done
```

Comment:
this too, `all_done` is intentional to make sure that the cluster is deleted irrespective of whatever happened upstream (e.g. a failed job should not prevent a cluster from being deleted)